base/applications/calc/utl_ieee.c

```diff
@@ -27,7 +27,7 @@ void prepare_rpn_result_2(calc_number_t *rpn, TCHAR *buffer, int size, int base)
 
     switch (base) {
     case IDC_RADIO_HEX:
-        StringCbPrintf(buffer, sizeof(buffer), _T("%I64X"), rpn->i);
+        StringCchPrintf(buffer, size, _T("%I64X"), rpn->i);
```

Comment:
OK this is better. And I confirm that `size` is indeed a count in characters.
Because: prepare_rpn_result_2 is called by prepare_rpn_result, and that latter one is called with SIZEOF e.g. like this:
`prepare_rpn_result(rpn, calc.buffer, SIZEOF(calc.buffer), calc.base);`

Reply:
Oh by the way, I had to double-check: that "SIZEOF" macro in calc.h is really like an ARRAYSIZE or `_countof` . Its naming causes confusion.